spec/models/note_spec.rb

```diff
@@ -1,6 +1,158 @@
 require 'rails_helper'
 
 RSpec.describe Note, type: :model do
-  it do
+  subject do
+    create(:note)
+  end
+
+  let(:user) { create(:user, utility: utility) }
+  let(:note) { build(:note, note_type: :review, content: Faker::Lorem.words(number: content), user: user) }
+
+  describe 'validations' do
+    context 'when validating presence' do
+      %i[user_id title content note_type].each do |value|
+        it { is_expected.to validate_presence_of(value) }
+      end
+    end
+
+    context 'when user has NorthUtility' do
+      let(:utility) { create(:north_utility) }
+
+      context 'with valid word count' do
+        let(:content) { rand(0..50) }
+
+        it 'is valid if review word count is in NorthUtility limit' do
+          expect(note).to be_valid
+        end
```

Comment:
Acá como estaríamos probando el `save!` es necesario persistir la nota en la DB. Tu `it` deberia verse algo mas como:
```suggestion
        it 'create note succesfully' do
          expect(note.save!).to be_truthy
        end
```
Esto es ya que queremos guardar la nota para que se ejecuten las validaciones, recorda que creando la instancia no es suficiente para esto

Reply:
Cambie el it para que sea mas conciso como dijiste